src/components/table/index.tsx

```diff
@@ -201,7 +206,7 @@ export default function EnhancedTable<T>({
   return (
     <>
       {showTopBar && <EnhancedTableToolbar numSelected={selectedRows ? selectedRows.length : 0} topBarButtons={topBarButtons} />}
-      <TableContainer id={id}>
+      <TableContainer id={id} ref={containerRef}>
         <DndContext sensors={sensors} onDragEnd={handleDragEnd}>
           <Table stickyHeader={true} aria-labelledby='tableTitle' size='medium' aria-label='enhanced table' className={classes.table}>
```

Comment:
I tried forever to get the built-in `stickyHeader` functionality to work. The best I could do involved fixing the height of the table container, which produced a gross double-scrollbar effect.